TacTok/count_names.py

```diff
@@ -33,6 +34,12 @@ def incr_ident(ident):
     else:
         idents[ident] += 1
 
+def is_name(node):
+    if get_locals:
+        return node.data == 'constructor_var' or node.data == 'constructor_name'
```

Comment:
Hmm, how does this include local variables or global definitions? Does "constructor_var" mean something broader than constructors?

Reply:
There are only for global definitions, not locals. This is just the script to dump everything to a vocab, so the switch determines whether it's the local or global vocab. So we don't need the names___id__t nodes; we just run it separately for each vocab. It runs once and is done forever, I just kept it there for reproducibility and easy extension.